portal/src/components/UU/uu.scss

```diff
@@ -0,0 +1,116 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+@import "~@fremtind/jkl-core/mixins/_all.scss";
+
+.jkl-portal__uu {
+    h2 {
+        @include jkl-text-style("desktop/title-small");
+    }
+    h3 {
+        @include jkl-text-style("desktop/heading-large");
+    }
+    h4 {
+        @include jkl-text-style("desktop/heading-medium");
+    }
+
+    .uu__filter-result-list {
+        list-style: none;
+        display: flex;
+        flex-wrap: wrap;
+        column-gap: 18px;
+        row-gap: 21px;
+        padding-inline-start: 0;
+        max-width: rem(750px);
+
+        @include jkl-text-style("desktop/body");
+    }
+
+    article {
+        margin-bottom: $layout-spacing--xl;
+    }
+
+    .uu__filter-search {
+        margin-top: $layout-spacing--large;
+    }
+
+    .uu__section--search {
+        position: relative;
+        z-index: 0;
+        margin-top: $layout-spacing--large;
+        padding-top: $layout-spacing--medium;
+        padding-bottom: $layout-spacing--xl;
+
+        & > *:not(:first-child) {
+            margin-top: $layout-spacing--small;
+        }
+
+        &::before {
+            content: "";
+            z-index: -1;
+            position: absolute;
+            top: 0;
+            left: -40px;
+            height: 100%;
+            width: 100vw;
+            background-color: $gra-10;
+
+            @include from-large-device {
+                left: -415px;
```

Comment:
This is one hell of a random value... Tar den gjerne som en variabel, og om den er knyttet til en annen verdi, gjerne samme.

Reply:
Ryddet opp i stylingen av bakgrunnen og lagt til en kommentar om hvor 415px kommer fra. Har også fikset scrollbar problematikken (heia macos som gjemmer scrollbaren by default).